File: apps/java/VehicleDetectorApp.java
import java.nio.file.Path;
import java.nio.file.Paths;
import java.io.IOException;
import org.sertiscorp.oneml.alpr.*;

public class VehicleDetectorApp {

    public static void main(String[] args) throws IOException {
        Path basePath = Paths.get("", "../../assets/images");
        String path = basePath + "/vehicle-detect-set/vehicle/cars.jpg";

        LicenseManager manager = new LicenseManager();
        manager.activateTrial();

        VehicleDetector detector = new VehicleDetector(manager);
        Utils utils = new Utils(manager);
        Image img = utils.readImageCV(path);

        VehicleDetectorResult result = detector.detect(img);

        System.out.println("Vehicles: " + result.getSize());

        ScoreList scores = result.getScores();
        BBoxList bboxes = result.getBBoxes();

        for (int i = 0; i < result.getSize(); i++) {
            System.out.println("Vehicle " + i);
            System.out.println(String.format("Score: %.6f", scores.get(i)));

            BBox box = bboxes.get(i);
            String out = String.format("BBox[top=%.6f,left=%.6f,bottom=%.6f,right=%.6f]", box.getTop(), box.getLeft(), box.getBottom(), box.getRight());
            System.out.println(out);
        }


    }
}

File: apps/csharp/VehicleDetectorApp.cs
using OneML.Alpr;

class VehicleDetectorApp {
    static void Main(string[] args) {
        LicenseManager manager = new LicenseManager();
        manager.ActivateTrial();

        using (VehicleDetector detector = new VehicleDetector(manager))
        using (Utils utils = new Utils(manager))
        {
            string basePath = @"../../assets/images/";
            string path = basePath + @"vehicle-detect-set/vehicle/cars.jpg";

            Image img = utils.ReadImageCV(path);

            VehicleDetectorResult result = detector.Detect(img);
            Console.WriteLine("Vehicles: " + result.GetSize());

            ScoreList scores = result.GetScores();
            BBoxList bboxes = result.GetBBoxes();
            StatusCodeList statuses = result.GetReturnStatus();

            string precision = "F6";
            for (int i = 0; i < result.GetSize(); i++) {
                Console.WriteLine("Vehicle " + i);
                Console.WriteLine("status: " + statuses[i]);
                Console.WriteLine("Score: " + scores[i].ToString(precision));

                BBox box = bboxes[i];
                string print = "BBox[top=" + box.top.ToString(precision) + ",left=" + box.left.ToString(precision) + ",bottom=" + box.bottom.ToString(precision) + ",right=" + box.right.ToString(precision) + "]";
                Console.WriteLine(print);
            }

        }
    }
}